include/mxnet/c_api.h

```diff
@@ -51,6 +51,10 @@ typedef void *RecordIOHandle;
 /*! \brief handle to MXRtc*/
 typedef void *RtcHandle;
 
+namespace mxnet {
+class NDArray;
```

Comment:
how about change to struct ndarray？
On Nov 23, 2015 6:22 PM, "Tianqi Chen" <notifications@github.com> wrote:

> In include/mxnet/c_api.h
> <https://github.com/dmlc/mxnet/pull/689#discussion_r45690557>:
>
> > @@ -51,6 +51,10 @@ typedef void *RecordIOHandle;
> >  /*! \brief handle to MXRtc*/
> >  typedef void *RtcHandle;
> >
> > +namespace mxnet {
> > +class NDArray;
>
> This is a C API, so please avoid include C++ object here, otherwise it can
> break for some FFI
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dmlc/mxnet/pull/689/files#r45690557>.
>